kairon/chat/handlers/channels/messenger.py

```
@@ -185,10 +194,16 @@ async def _handle_user_message(
-            username = await out_channel.get_username_for_id(sender_id)
-            if username and (username not in self.allowed_users):
-                return
+
+        if self.is_instagram:
+            if self.allowed_users and isinstance(self.allowed_users, list):
+                username = await out_channel.get_username_for_id(sender_id)
+                if username not in self.allowed_users:
+                    return
+
+            if metadata.get("media_id") in self.post_config:
+                if text.lower() not in map(str.lower, metadata.get("keywords")):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)